src/Models/Concerns/HasConversations.php

```diff
@@ -39,4 +41,19 @@ public function readMessages(): BelongsToMany
         return $this->belongsToMany(config('gossip.models.message'))
             ->withTimestamps();
     }
+
+    /**
+     * Scope a query to only include users of the given conversation.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder  $query
+     * @param  int  $conversationId
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeOfConversation(Builder $query, int $conversationId): Builder
```

Comment:
Personally I would prefer to send the whole `Model` to these kind of methods. Maintains flexibility.